src/transforms/log_to_metric.rs

```diff
@@ -35,6 +48,35 @@ const ORIGIN_SERVICE_VALUE: u32 = 3;
 pub struct LogToMetricConfig {
     /// A list of metrics to generate.
     pub metrics: Vec<MetricConfig>,
+    /// Setting this flag will change the behavior of this transformation.
+    /// Notably the `metrics` field will be ignored. 
+    /// All incoming events will be processed and if possible they will be converted to log events. 
+    /// Otherwise, only items specified in the 'metrics' field will be processes.
```

Comment:
```suggestion
    /// Notably the `metrics` field is ignored. 
    /// All incoming events are processed and, if possible, are converted to log events. 
    /// Otherwise, only items specified in the 'metrics' field are processed.
```
I'm guessing this should be 'processed', not 'processes'